pyproject.toml

```
@@ -14,6 +14,7 @@ requires-python = ">=3.13,<3.14"
 dependencies = [
   "aiohttp==3.10.9",
   "arrow==1.3.0",
+  "audioop-lts>=0.2.2",
```

###### ❓ Question

Should we pin this?

```suggestion
  "audioop-lts==0.2.2",
```